nbs/models.ipynb

```diff
@@ -57,6 +57,7 @@
     "import pandas as pd\n",
```

Comment:
<p>It would be great to add a unitary test, something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">fcst_ha = ha.forecast(ap, 12, None, None, False, (80,95))
test_eq(
    fcst_ha['lo-80'],
    #here goes the expected value for lo-80
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='90'/>

Reply:
<p>Used <code>np.testing.assert_almost_equal(... decimal=4)</code> instead since Hyndman's functions round results up to 4 decimals and Python doesn't, </p>